README.md

```diff
@@ -0,0 +1,9 @@
+# shar
+Yet another tool for video streaming (WIP)
```

Comment:
I would like to see here good explanation of what the shar is (cross-platform p2p screen sharing). I'd suggest to ask @0xd34d10cc about this